gradle.properties

```diff
@@ -0,0 +1,23 @@
+android.enableJetifier=true
```

Comment:
we don't need this, do we ?

Reply:
It's necessary for the sample project for now unfortunately, greenrobot's libraries still think it's 2018.